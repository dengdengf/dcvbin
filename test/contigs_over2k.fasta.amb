176587 66 0
