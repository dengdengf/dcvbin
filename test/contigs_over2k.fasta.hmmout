#                                                                            --- full sequence --- -------------- this domain -------------   hmm coord   ali coord   env coord
# target name        accession   tlen query name           accession   qlen   E-value  score  bias   #  of  c-Evalue  i-Evalue  score  bias  from    to  from    to  from    to  acc description of target
#------------------- ---------- ----- -------------------- ---------- ----- --------- ------ ----- --- --- --------- --------- ------ ----- ----- ----- ----- ----- ----- ----- ---- ---------------------
NODE_29_length_2426_cov_1.513286_589_795_- -             68 TIGR00001            TIGR00001     63   1.8e-22   73.5   9.9   1   1   7.4e-25     2e-22   73.3   9.9     2    63     5    66     4    66 0.99 -
NODE_29_length_2426_cov_1.513286_823_1254_- -            143 TIGR00168            TIGR00168    165     2e-58  190.7   2.2   1   1   8.4e-61   2.3e-58  190.5   2.2    24   163     2   140     1   142 0.98 -
NODE_60_length_2085_cov_1.325123_1_1464_- -            488 TIGR00418            TIGR00418    565  5.7e-160  527.8   1.7   1   1  2.4e-162  6.4e-160  527.7   1.7     1   468    23   488    23   488 0.98 -
NODE_29_length_2426_cov_1.513286_177_530_- -            117 TIGR01032            TIGR01032    114   2.8e-50  163.4   6.4   1   1   1.2e-52   3.2e-50  163.2   6.4     1   114     1   114     1   114 0.99 -
#
# Program:         hmmsearch
# Version:         3.3 (Nov 2019)
# Pipeline mode:   SEARCH
# Query file:      /media/ubuntu/abc/csm/DCVBin_project/marker_gene/auxiliary/marker.hmm
# Target file:     /media/ubuntu/abc/csm/DCVBin_project/test/contigs_over2k.fasta.frag.faa
# Option settings: /media/ubuntu/abc/csm/DCVBin_project/marker_gene/auxiliary/hmmer-3.3/src/hmmsearch --domtblout /media/ubuntu/abc/csm/DCVBin_project/test/contigs_over2k.fasta.hmmout --cut_tc --cpu 40 /media/ubuntu/abc/csm/DCVBin_project/marker_gene/auxiliary/marker.hmm /media/ubuntu/abc/csm/DCVBin_project/test/contigs_over2k.fasta.frag.faa 
# Current dir:     /media/ubuntu/abc/csm/DCVBin_project
# Date:            Tue Jun 17 17:55:24 2025
# [ok]
