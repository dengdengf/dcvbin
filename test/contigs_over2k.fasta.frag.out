>NODE_1_length_6080_cov_2.040664
1	252	-	1	1.307865	I:	D:
407	832	-	2	1.331126	I:	D:
873	1265	-	3	1.352619	I:	D:
1386	1607	+	3	1.402445	I:	D:
1824	1976	-	3	1.364583	I:	D:
2273	2713	+	2	1.377216	I:	D:
2942	3211	-	2	1.324963	I:	D:
3295	3792	+	1	1.340495	I:	D:
3840	4529	+	3	1.319233	I:	D:
4649	5770	+	2	1.313130	I:	D:
5884	6078	+	1	1.306956	I:	D:
>NODE_2_length_5271_cov_1.936925
2	727	-	2	1.256529	I:	D:
730	930	-	1	1.278730	I:	D:
1088	2488	-	2	1.313677	I:	D:
2493	3500	-	3	1.316353	I:	D:
3513	4466	-	3	1.292428	I:	D:
4589	5101	-	2	1.427755	I:	D:
>NODE_3_length_5084_cov_1.639093
1	1593	-	1	1.329625	I:	D:
1604	2092	-	2	1.342177	I:	D:
2175	2579	-	3	1.362767	I:	D:
2590	2856	-	1	1.314069	I:	D:
2897	3421	-	2	1.340299	I:	D:
3477	3728	-	3	1.314350	I:	D:
3745	4260	-	1	1.329237	I:	D:
4272	5081	-	3	1.327637	I:	D:
>NODE_4_length_4906_cov_1.719027
1	1743	-	1	1.334139	I:	D:
1765	2079	-	1	1.341005	I:	D:
2599	2736	-	1	1.342217	I:	D:
2808	2927	+	3	1.311609	I:	D:
2995	3291	-	1	1.349145	I:	D:
3418	4905	-	1	1.308221	I:	D:
>NODE_5_length_4187_cov_3.133349
107	304	-	2	1.325649	I:	D:
315	560	-	3	1.331455	I:	D:
587	838	-	2	1.320622	I:	D:
879	1010	-	3	1.303265	I:	D:
1251	2276	+	3	1.345164	I:	D:
2313	2543	-	3	1.333115	I:	D:
2563	2898	-	1	1.345086	I:	D:
2972	3982	-	2	1.368381	I:	D:
>NODE_6_length_4105_cov_1.659012
1	642	+	1	1.307823	I:	D:
647	877	+	2	1.326464	I:	D:
898	2898	+	1	1.332714	I:	D:
2939	3166	+	2	1.361456	I:	D:
3177	3545	+	3	1.320272	I:	D:
3761	3874	+	2	1.362116	I:	D:
4009	4104	+	1	1.356265	I:	D:
>NODE_7_length_3676_cov_1.506490
1	1413	-	1	1.322918	I:	D:
1469	1630	-	2	1.284194	I:	D:
1710	2060	-	3	1.348901	I:	D:
2089	2325	-	1	1.405500	I:	D:
2536	2991	-	1	1.317086	I:	D:
3022	3240	+	1	1.355681	I:	D:
3398	3673	+	2	1.334978	I:	D:
>NODE_8_length_3554_cov_3.452415
316	387	+	1	1.319707	I:	D:
684	779	+	3	1.371790	I:	D:
1380	1538	+	3	1.395258	I:	D:
1695	1856	+	3	1.372927	I:	D:
2357	2428	+	2	1.343747	I:	D:
2487	2711	+	3	1.405469	I:	D:
2759	2941	-	2	1.393186	I:	D:
3156	3551	-	3	1.409405	I:	D:
>NODE_9_length_3536_cov_1.941396
3	146	-	3	1.351805	I:	D:
195	371	-	3	1.386345	I:	D:
552	1802	+	3	1.339433	I:	D:
1872	2642	-	3	1.301827	I:	D:
2734	3195	+	1	1.299503	I:	D:
3199	3534	-	1	1.294311	I:	D:
>NODE_10_length_3358_cov_1.706630
1	609	+	1	1.304317	I:	D:
834	2408	+	3	1.342232	I:	D:
2469	2684	+	3	1.365205	I:	D:
2779	3357	+	1	1.362024	I:	D:
>NODE_11_length_3123_cov_2.091265
2	340	-	2	1.337113	I:	D:
347	1402	-	2	1.325347	I:	D:
1427	2446	-	2	1.299277	I:	D:
2470	3120	-	1	1.302110	I:	D:
>NODE_12_length_3049_cov_2.903474
2	118	-	2	1.418993	I:	D:
456	2129	-	3	1.354520	I:	D:
2205	2570	-	3	1.345413	I:	D:
2574	2918	-	3	1.368530	I:	D:
>NODE_13_length_2995_cov_1.331633
2	79	-	2	1.388244	I:	D:
181	441	-	1	1.429337	I:	D:
450	779	-	3	1.340229	I:	D:
1128	1472	+	3	1.351760	I:	D:
1497	2045	+	3	1.310802	I:	D:
2083	2667	+	1	1.351786	I:	D:
2809	2994	+	1	1.358071	I:	D:
>NODE_14_length_2956_cov_1.525681
1	276	+	1	1.296109	I:	D:
316	1026	+	1	1.282769	I:	D:
1050	1121	+	3	1.325436	I:	D:
1182	1328	+	3	1.345741	I:	D:
1533	2213	+	3	1.279523	I:	D:
2355	2693	+	3	1.261357	I:	D:
2699	2953	+	2	1.262366	I:	D:
>NODE_15_length_2878_cov_2.685795
15	146	-	3	1.230086	I:	D:
225	293	-	3	1.197352	I:	D:
401	1960	-	2	1.304202	I:	D:
1996	2676	-	1	1.266700	I:	D:
2690	2875	-	2	1.445461	I:	D:
>NODE_16_length_2878_cov_1.701382
1	1107	-	1	1.278436	I:	D:
1280	2875	+	2	1.283288	I:	D:
>NODE_17_length_2859_cov_1.774608
1	1317	-	1	1.297360	I:	D:
1394	1828	-	2	1.316132	I:	D:
1843	2712	-	1	1.324608	I:	D:
2733	2801	-	3	1.238350	I:	D:
>NODE_18_length_2849_cov_1.479241
3	590	-	3	1.266387	I:	D:
654	1580	-	3	1.260512	I:	D:
1731	2582	-	3	1.367307	I:	D:
>NODE_19_length_2755_cov_2.440000
3	1301	+	3	1.321690	I:	D:
1351	2124	-	1	1.333213	I:	D:
2415	2753	-	3	1.344644	I:	D:
>NODE_20_length_2722_cov_1.845894
3	737	+	3	1.328212	I:	D:
1006	2379	+	1	1.311823	I:	D:
2504	2719	+	2	1.280239	I:	D:
>NODE_21_length_2606_cov_1.620541
2	571	+	2	1.310945	I:	D:
595	888	+	1	1.284226	I:	D:
922	1758	+	1	1.302683	I:	D:
1796	2125	-	2	1.302761	I:	D:
2170	2412	-	1	1.302916	I:	D:
2473	2604	-	1	1.329343	I:	D:
>NODE_22_length_2603_cov_1.696625
2	1369	-	2	1.308047	I:	D:
1449	1835	-	3	1.294958	I:	D:
1880	2500	-	2	1.313045	I:	D:
>NODE_23_length_2581_cov_1.866192
3	572	+	3	1.260420	I:	D:
866	1435	-	2	1.297993	I:	D:
1438	2007	-	1	1.281688	I:	D:
2133	2294	+	3	1.356426	I:	D:
>NODE_24_length_2547_cov_1.797753
2	142	-	2	1.353486	I:	D:
159	989	-	3	1.361038	I:	D:
1421	1636	-	2	1.325156	I:	D:
1737	1979	+	3	1.386112	I:	D:
2053	2127	+	1	1.306273	I:	D:
2138	2395	+	2	1.382571	I:	D:
2427	2546	+	3	1.383358	I:	D:
>NODE_25_length_2525_cov_1.420243
1	1041	-	1	1.255828	I:	D:
1059	1892	-	3	1.244150	I:	D:
1910	2524	-	2	1.273288	I:	D:
>NODE_26_length_2500_cov_1.544785
1	273	-	1	1.277697	I:	D:
837	1955	+	3	1.301821	I:	D:
1958	2497	+	2	1.341651	I:	D:
>NODE_27_length_2499_cov_5.259820
3	287	+	3	1.328056	I:	D:
529	726	+	1	1.334481	I:	D:
733	2397	+	1	1.324337	I:	D:
>NODE_28_length_2439_cov_1.376678
3	137	+	3	1.311736	I:	D:
417	1691	+	3	1.326481	I:	D:
1770	1895	+	3	1.295121	I:	D:
2127	2438	-	3	1.362864	I:	D:
>NODE_29_length_2426_cov_1.513286
2	154	-	2	1.277673	I:	D:
177	530	-	3	1.280658	I:	D:
589	795	-	1	1.291861	I:	D:
823	1254	-	1	1.268903	I:	D:
1632	2015	-	3	1.283463	I:	D:
2055	2423	+	3	1.370455	I:	D:
>NODE_30_length_2418_cov_1.927211
232	861	+	1	1.340589	I:	D:
871	2415	+	1	1.294100	I:	D:
>NODE_31_length_2399_cov_1.647184
107	385	+	2	1.372410	I:	D:
457	834	+	1	1.346689	I:	D:
868	1140	+	1	1.310113	I:	D:
1184	1432	+	2	1.376801	I:	D:
1463	1639	+	2	1.339861	I:	D:
1729	2397	+	1	1.314970	I:	D:
>NODE_32_length_2399_cov_1.491041
2	1471	-	2	1.262857	I:	D:
1676	2398	+	2	1.268499	I:	D:
>NODE_33_length_2392_cov_1.881900
107	265	-	2	1.301816	I:	D:
408	2279	-	3	1.337815	I:	D:
>NODE_34_length_2378_cov_1.622040
3	416	-	3	1.322268	I:	D:
561	1589	+	3	1.282737	I:	D:
1799	2377	-	2	1.342798	I:	D:
>NODE_35_length_2345_cov_1.265066
182	343	+	2	1.267412	I:	D:
348	668	+	3	1.374071	I:	D:
761	1738	+	2	1.352227	I:	D:
1851	1994	+	3	1.318286	I:	D:
2038	2271	+	1	1.306839	I:	D:
>NODE_36_length_2335_cov_1.498246
2	460	-	2	1.262777	I:	D:
1019	2005	-	2	1.272169	I:	D:
2027	2332	-	2	1.262072	I:	D:
>NODE_37_length_2317_cov_1.909372
1	2316	+	1	1.299748	I:	D:
>NODE_38_length_2301_cov_4.853963
479	1765	-	2	1.361094	I:	D:
1938	2300	+	3	1.372165	I:	D:
>NODE_39_length_2251_cov_2.248179
1	273	+	1	1.355902	I:	D:
288	509	+	3	1.423471	I:	D:
688	1047	+	1	1.401784	I:	D:
1279	2010	-	1	1.279410	I:	D:
2011	2250	-	1	1.361832	I:	D:
>NODE_40_length_2249_cov_1.600273
1	195	+	1	1.419208	I:	D:
218	1387	+	2	1.255508	I:	D:
1594	2247	+	1	1.255697	I:	D:
>NODE_41_length_2210_cov_2.051508
2	367	+	2	1.304381	I:	D:
412	771	+	1	1.323740	I:	D:
783	1052	+	3	1.366598	I:	D:
1212	1283	+	3	1.338470	I:	D:
1420	2208	-	1	1.318475	I:	D:
>NODE_42_length_2209_cov_1.770195
1	177	-	1	1.337648	I:	D:
322	690	+	1	1.316690	I:	D:
718	1017	-	1	1.335250	I:	D:
1034	1387	-	2	1.339444	I:	D:
1414	2208	-	1	1.313152	I:	D:
>NODE_43_length_2197_cov_1.286181
3	182	-	3	1.265203	I:	D:
199	717	-	1	1.258723	I:	D:
1361	1993	+	2	1.235021	I:	D:
2069	2164	+	2	1.297410	I:	D:
>NODE_44_length_2184_cov_1.639267
2	349	+	2	1.268312	I:	D:
587	697	-	2	1.263690	I:	D:
864	965	-	3	1.252606	I:	D:
1072	2181	+	1	1.261364	I:	D:
>NODE_45_length_2182_cov_1.528444
3	1574	+	3	1.309330	I:	D:
1696	2181	+	1	1.332984	I:	D:
>NODE_46_length_2178_cov_1.648610
3	2177	-	3	1.327295	I:	D:
>NODE_47_length_2174_cov_1.391694
2	202	+	2	1.314153	I:	D:
402	572	+	3	1.356365	I:	D:
586	768	+	1	1.358404	I:	D:
774	1871	+	3	1.342285	I:	D:
1875	2171	+	3	1.372959	I:	D:
>NODE_48_length_2172_cov_5.982522
1	108	+	1	1.393885	I:	D:
736	891	+	1	1.376511	I:	D:
1162	1605	+	1	1.398930	I:	D:
>NODE_49_length_2165_cov_1.647867
120	278	-	3	1.340718	I:	D:
314	2164	-	2	1.345513	I:	D:
>NODE_50_length_2159_cov_4.291350
2	340	+	2	1.389356	I:	D:
352	678	+	1	1.354593	I:	D:
710	967	+	2	1.305546	I:	D:
1059	1217	+	3	1.365400	I:	D:
1330	1542	-	1	1.346484	I:	D:
1577	2158	+	2	1.356229	I:	D:
>NODE_51_length_2157_cov_1.516651
2	700	-	2	1.237335	I:	D:
731	1477	-	2	1.239688	I:	D:
1490	2155	-	2	1.271306	I:	D:
>NODE_52_length_2147_cov_1.439293
70	2144	+	1	1.291478	I:	D:1784,
>NODE_53_length_2143_cov_2.038793
1	2142	-	1	1.321769	I:	D:
>NODE_54_length_2142_cov_1.489219
2	361	+	2	1.311425	I:	D:
515	682	-	2	1.323119	I:	D:
703	2139	+	1	1.303493	I:	D:
>NODE_55_length_2138_cov_1.508881
2	1249	-	2	1.275700	I:	D:
1416	2135	-	3	1.247359	I:	D:
>NODE_56_length_2138_cov_1.285646
67	462	-	1	1.315688	I:	D:
498	1691	-	3	1.289617	I:	D:
1755	2135	-	3	1.315562	I:	D:
>NODE_57_length_2135_cov_1.588462
2	463	-	2	1.268375	I:	D:
818	2134	-	2	1.235243	I:	D:
>NODE_58_length_2098_cov_1.904552
1	1002	-	1	1.346070	I:	D:
1092	1268	-	3	1.327535	I:	D:
1402	1671	-	1	1.337779	I:	D:
1783	2055	+	1	1.373989	I:	D:
>NODE_59_length_2085_cov_1.466995
3	293	+	3	1.317355	I:	D:
376	528	+	1	1.318117	I:	D:
536	673	+	2	1.355241	I:	D:
761	1111	+	2	1.350755	I:	D:
1198	2082	+	1	1.331252	I:	D:
>NODE_60_length_2085_cov_1.325123
1	1464	-	1	1.260060	I:	D:
1636	1752	-	1	1.274484	I:	D:
1951	2082	-	1	1.338143	I:	D:
>NODE_61_length_2073_cov_2.346878
1	507	-	1	1.341154	I:	D:
799	1257	-	1	1.369673	I:	D:
1298	1690	-	2	1.391914	I:	D:
1827	2072	-	3	1.298586	I:	D:
>NODE_62_length_2027_cov_1.471602
3	1262	-	3	1.337388	I:	D:
1284	2024	-	3	1.337758	I:	D:
>NODE_63_length_2023_cov_1.603659
2	808	+	2	1.342266	I:	D:
956	2020	-	2	1.342685	I:	D:
>NODE_64_length_2018_cov_4.795721
250	384	+	1	1.368338	I:	D:
458	1864	+	2	1.378194	I:	D:
1943	2017	-	2	1.300571	I:	D:
>NODE_65_length_2015_cov_1.849490
2	793	-	2	1.315301	I:	D:
823	1680	-	1	1.336288	I:	D:
1766	1855	-	2	1.289321	I:	D:
1927	2013	+	1	1.307441	I:	D:
>NODE_66_length_2002_cov_1.838213
2	175	+	2	1.411113	I:	D:
280	384	-	1	1.332005	I:	D:
441	731	+	3	1.389837	I:	D:
741	851	-	3	1.301766	I:	D:
1079	1870	-	2	1.323652	I:	D:
